actors/user/src/test/java/org/sunbird/user/UserRoleActorTest.java

```diff
@@ -0,0 +1,252 @@
+package org.sunbird.user;
+
+import static akka.testkit.JavaTestKit.duration;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSelection;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.testkit.javadsl.TestKit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.actor.router.RequestRouter;
+import org.sunbird.actor.service.BaseMWService;
+import org.sunbird.actorutil.InterServiceCommunication;
+import org.sunbird.actorutil.InterServiceCommunicationFactory;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.models.util.datasecurity.DecryptionService;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.dto.SearchDTO;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.actors.role.dao.impl.RoleDaoImpl;
+import org.sunbird.learner.util.Util;
+import org.sunbird.user.actors.UserRoleActor;
+import org.sunbird.user.dao.UserOrgDao;
+import org.sunbird.user.dao.impl.UserOrgDaoImpl;
+import scala.concurrent.duration.Duration;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  RoleDaoImpl.class,
+  BaseMWService.class,
+  ActorSelection.class,
+  ActorRef.class,
+  RequestRouter.class,
+  InterServiceCommunicationFactory.class,
+  ElasticSearchUtil.class,
+  Util.class,
+  UserOrgDaoImpl.class,
+  DecryptionService.class,
+})
+@PowerMockIgnore({"javax.management.*"})
+public class UserRoleActorTest {
+
+  private ActorSystem system = ActorSystem.create("system");
+
+  private final Props props = Props.create(UserRoleActor.class);
+  private CassandraOperationImpl cassandraOperation;
+  private SearchDTO searchDTO;
+  private static InterServiceCommunication interServiceCommunication =
+      Mockito.mock(InterServiceCommunication.class);
+
+  private Response response = Mockito.mock(Response.class);
+  private static DecryptionService decryptionService;
```

Comment:
Please check if above variables are required
if required are they static
check scope & life-cycle for each above variable